src/test/java/seedu/address/model/tag/LevelSubjectCodeTest.java

```diff
@@ -6,16 +6,23 @@
 import org.junit.jupiter.api.Test;
 
 class LevelSubjectCodeTest {
-
     @Test
     public void testIsValidTag() {
         assertEquals(true, LevelSubjectCode.isValidTag(VALID_TAG_PM));
         assertEquals(false, LevelSubjectCode.isValidTag("ABCDE"));
```

Comment:
Perhaps you could make use of the `INVALID_TAG` in the CommandTestUtil.java file